core/src/wallet/tezos/explorers/ExternalTezosLikeBlockchainExplorer.cpp

```diff
@@ -34,6 +34,10 @@
 
 namespace ledger {
     namespace core {
+        namespace {
+            const std::string BCD_API_ENDPOINT = "https://api.better-call.dev/v1";
```

Comment:
hardcoded endpoint ?

Reply:
I am not sure how to do it differently (except defining it in a djinni file). If I understand correctly, the endpoint in the configuration will point to TZSTATS, so I cannot overwrite it otherwise the other functions will not work anymore

Reply:
We should not change the endpoint for other functions.
But, we may define a specific endpoint in configuration for getTokenBalance.
By the way, Is this functionality not provided by the current tzstats api ?

Reply:
I did not find what I needed with tzstats, and I've been told that the tezos foundation recommanded us to use BCD to deal with tokens.